frequency.rb

```diff
@@ -0,0 +1,14 @@
+#!/urs/bin/ruby -w
+
+hash = Hash.new
+str = String.new("aabbbcddeac")
+
+str.each_char(){ |c|
+	if hash.has_key?(c)
+		hash[c] = hash[c]+1;
+	else
+		hash[c] = 1;
+	end
+}
```

Comment:
1. For multi-line code use `do..end` syntax
2. Use spaces around operator
3. No need to use `()` with `each_char`
4. Use significant variable name instead of `c`